.ci/on-schedule.sh

```diff
@@ -124,7 +180,9 @@ function update_pkgbuild() {
     local PKGBUILD_SOURCE="${VARIABLES_UPDATE_PKGBUILD[CI_PKGBUILD_SOURCE]}"
 
     # Check if the package is from the AUR
-    if [[ "$PKGBUILD_SOURCE" != aur ]]; then
+    if [[ "$PKGBUILD_SOURCE" =~ ^gitlab:.*$ ]]; then
+        update_via_gitlab_api VARIABLES_UPDATE_PKGBUILD
+    elif [[ "$PKGBUILD_SOURCE" != aur ]]; then
```

Comment:
For now, I think this should not be implemented in this same PR

```suggestion
    if [[ "$PKGBUILD_SOURCE" != aur ]]; then
```